securedrop_export/disk/service.py

```diff
@@ -0,0 +1,133 @@
+import logging
+
+from securedrop_export.exceptions import ExportException
+
+from .cli import CLI
+from .status import Status
+from .new_status import Status as NewStatus
+
+logger = logging.getLogger(__name__)
+
+
+class Service:
+    def __init__(self, submission, cli=None):
+        self.submission = submission
+        self.cli = cli or CLI()
+
```

Comment:
by default, use the provided CLI, but can optionally pass in a different CLI implementation (eg for testing)